web/src/redux/apis/TraceTest.api.ts

```diff
@@ -10,6 +12,9 @@ import {TAssertion, TAssertionResults, TRawAssertionResults} from '../../types/A
 import {TRawTestDefinition, TTestDefinition} from '../../types/TestDefinition.types';
 import {TRawTestRun, TTestRun} from '../../types/TestRun.types';
 
+const webSocketGateway = WebSocketGateway({url: 'ws://localhost:8081/ws'});
```

Comment:
I think we should connect right away, maybe we can even connect directly from the gateway already, then every time it gets imported we just use the methods.

I don't think we'll be connecting to any other web socket in the near future.